src/uploader.ts

```diff
@@ -153,10 +181,10 @@ export default class Uploader {
        */
       const formData = new FormData();
 
-      formData.append(this.config.field, file);
+      formData.append(this.config.field || '', file);
```

Comment:
field can't be `''`

Reply:
I guess it can be replaced with `'image'` instead of `''`

Reply:
yes, it would be right, since it was used in previous code